README.md

```diff
@@ -80,9 +80,9 @@ Change directories to the newly cloned repo:
 
      cd cordapp-tutorial
      
-Check out the latest milestone release (currently `release-M8.2`):
+Check out the latest milestone release (currently `release-M9.2`):
```

Comment:
Wonder if we want to specify a release at all, or point people at https://github.com/corda/corda/tags ?